test/functional/apps/discover/_discover.js

```diff
@@ -1,15 +1,14 @@
+
+import expect from 'expect.js';
+
 import {
   bdd,
   scenarioManager,
-  common,
-  discoverPage,
-  settingsPage,
-  headerPage,
   esClient,
   elasticDump
 } from '../../../support';
 
-var expect = require('expect.js');
+import PageObjects from '../../../support/page_objects';
```

Comment:
I totally agree about [omitting words that don't mean anything](http://journal.stuffwithstuff.com/2016/06/16/long-names-are-long/), but since a Page Object is a specific type of pattern and the term is in common use, I think it's better to call it by that term so we can avoid adding to our internal lexicon. My litmus test is: if we were to onboard someone and they ask "What does Page do?", would the answer be simply to call it something else? In this case I think I'd probably just say, "Oh, that's a collection of Page Objects". :)